### ISSUE TITLE
Elite Sports Team Selector

### ISSUE THREAD

**Issue by lespabrandzulic82** [alptium/branko.spalovic#5]

**Comment by lespabrandzulic82** [comment]
hey @valentinacupac ,I have done this task, could you take a look and let me know if  I need to make some changes?

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82,

1. There's no need for + '\n', instead use System.out.println (because println means print line, just just have multiple of those statements)
2. Please check my comments regarding #4 
3. Please write the logic here in plain English too, so that I can verify if it corresponds to the program logic.

**Comment by lespabrandzulic82** [comment]
I have changed my code based on comments above. 
The logic is following: Based on information which each candidates enters(age, position in team, height, number of assistentions, number of jumps and average score per game) system accepts or rejects his application. There are some criteria for each position-for position play: player needs to be older than 25 years, shorter than 200 cm and needs to have more than 6 assistentions per game in last season;  for position small forward: player needs to be older than 20 years, taller than 190 cm and needs to have more than 12 points per game in last season; for position center: player needs to be older than 28 years, taller than 205 cm and needs to have more than 7 jumps per game in last season. @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, it's better now, but still some improvements:
1. In your logic above, you wrote for example "older than 25 years" which in code is written as "> 25". However, if you wanted it to be ">= 25" then the sentence should be "25 years or older". Please clarify which one you meant, this is really important later for requirements analysis.
2. Fix formatting "int position= sc.nextInt();" (space issue)
3. Fix double space "What is  your average number"
4. I still see '\n', please change to System.out.println
5. Regarding brackets () in your if statement, see https://github.com/alptium/valentina.cupac/wiki/03.-Hello-World

**Comment by lespabrandzulic82** [comment]
I have changed my code based on logic above, I put ">" for years now. Regarding other comments, it is corrected, too. @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82:
2. Space issue needs to be fixed in other parts where there are operators, e.g.  Scanner sc= (see https://github.com/alptium/valentina.cupac/wiki/03.-Hello-World comment 15)
6. Spelling namber, Unfortunatelly, asist
7. https://github.com/alptium/valentina.cupac/wiki/03.-Hello-World comment 48

**Comment by lespabrandzulic82** [comment]
In the basketball dictionary, it says assist, but I have put in the assistance's. What do you mean spelling number ? @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, thanks for the clarification and update. 

Remaining points:
* "Unfortunatelly" - incorrect spelling
* "asist" - incorrect spelling
* line 16: space around the operator = is missing

**Comment by lespabrandzulic82** [comment]
Ok, I corrected it. Can you see the version with  OOP, if it needs to be done differently? I had a dilemma about the scanner, should It go to the Player class maybe...? @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, correct now, closing ticket.

P.S. For OOP version, that should be a separate ticket.

**Comment by valentinajemuovic** [comment]
Hi @lespabrandzulic82,

I've added a new requirement for this task for everyone, add support for multiple candidates, so usign ArrayList.

Thanks,
Valentina

**Comment by lespabrandzulic82** [comment]
I have made array list for this project and merged two projects in one. Please take a look. Thanks. @valentinacupac

**Comment by valentinajemuovic** [comment]
@lespabrandzulic82, as can be seen here https://github.com/alptium/branko.spalovic there are now 3 projects for Elite Sports Selector, please re-check your merge.

**Comment by lespabrandzulic82** [comment]
now i have just one project !@valentinacupac
